src/NosCore.Controllers/Miniland/MinilandObjects/UseobjPacketHandler.cs

```diff
@@ -43,76 +43,77 @@ public UseobjPacketHandler(IMinilandProvider minilandProvider, IWarehouseHttpCli
             _warehouseHttpClient = warehouseHttpClient;
         }
 
-        public override async Task Execute(UseObjPacket useobjPacket, ClientSession clientSession)
+        public override async Task ExecuteAsync(UseObjPacket useobjPacket, ClientSession clientSession)
         {
             var miniland = _minilandProvider.GetMiniland(clientSession.Character.CharacterId);
             var minilandObject =
                 clientSession.Character.MapInstance.MapDesignObjects.Values.FirstOrDefault(s =>
                     s.Slot == useobjPacket.ObjectId);
-            if ((minilandObject != null) && (miniland != null))
+            if (minilandObject == null)
             {
-                if (!minilandObject.InventoryItemInstance!.ItemInstance!.Item!.IsWarehouse)
+                return;
+            }
+
+            if (!minilandObject.InventoryItemInstance!.ItemInstance!.Item!.IsWarehouse)
+            {
+                var game = (byte) (minilandObject.InventoryItemInstance.ItemInstance.Item.EquipmentSlot ==
+                    EquipmentType.MainWeapon
+                        ? (4 + minilandObject.InventoryItemInstance.ItemInstance.ItemVNum) % 10
+                        : (int) minilandObject.InventoryItemInstance.ItemInstance.Item.EquipmentSlot / 3);
+                var full = false;
+                await clientSession.SendPacketAsync(new MloInfoPacket
                 {
-                    var game = (byte) (minilandObject.InventoryItemInstance.ItemInstance.Item.EquipmentSlot ==
-                        EquipmentType.MainWeapon
-                            ? (4 + minilandObject.InventoryItemInstance.ItemInstance.ItemVNum) % 10
-                            : (int) minilandObject.InventoryItemInstance.ItemInstance.Item.EquipmentSlot / 3);
-                    var full = false;
-                    await clientSession.SendPacket(new MloInfoPacket
+                    IsOwner = miniland.MapInstanceId == clientSession.Character.MapInstanceId,
+                    ObjectVNum = minilandObject.InventoryItemInstance.ItemInstance.ItemVNum,
+                    Slot = (byte) useobjPacket.ObjectId,
+                    MinilandPoints = miniland.MinilandPoint,
+                    LawDurability = minilandObject.DurabilityPoint < 1000,
+                    IsFull = full,
+                    MinigamePoints = new MloInfoPacketSubPacket[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5266673)